code/game/objects/items/emags.dm

```diff
@@ -34,9 +41,16 @@
 	name = "cryptographic sequencer"
 	icon_state = "emag"
 	inhand_icon_state = "card-id"
+	slot_flags = ITEM_SLOT_ID
+	worn_icon_state = "emag"
```

Comment:
```suggestion
```

unless im missing something these should be inherited from the parent class

Reply:
look closely, it's /emagfake not /emag/fake

Reply:
oh sorry, i got pranked by github

![image](https://user-images.githubusercontent.com/10467687/109746834-6be9ae00-7bac-11eb-9d48-cfc2389eb9e9.png)

i thought for some reason you were changing ``/obj/item/card/emag/bluespace``

Reply:
fair enough, that does look very hard to see in dark mode